### ISSUE TITLE
a way of temporarily muting a user

### ISSUE THREAD

**Issue by epicEaston197** [Neos-Metaverse/NeosPublic#3429]
### Is your feature request related to a problem? Please describe.

during music events where people have to have their mics muted and sometimes people forget to mute their mic and if you silence them it makes it so they can't unmute their mic when they want to so having a temporary mute option would be would be great 

it would basically act like silence but the user can unmute whenever they want to

### Relevant issues

none that I could think of

### Describe the solution you'd like

a way of globally muting a user but have the user have an option of unmuting themselves whenever they want to

### Describe alternatives you've considered

none really except maybe silencing the user and having that user contact one of the moderators but this has its social down sides

**Comment by iamgreaser** [comment]
I *think* it's possible to change a user's voice mode to "Muted" via the User Editor (DevToolTip -> New -> Editors -> User Editor or something, I don't have Neos open right now) as I *think* I did exactly that to someone at one point, but I'm a bit hazy on the details.

**Comment by H3BO3** [comment]
@iamgreaser You can locally mute users through the Session menu. Easton is requesting something similar to Silencing which is usually only available with higher permission levels.

**Comment by iamgreaser** [comment]
@H3BO3 I'm aware of that, and that isn't what I was talking about, I was talking about forcing a user's voice mode to Muted via the User Editor.

**Comment by shadowpanther** [comment]
Which might stop working with hard permissions because you are intruding into other users' userspace.

**Comment by ProbablePrime** [comment]
Lets skip to talking about what you'd want our solution here to be rather than talking about in-game solutions.

**Comment by ProbablePrime** [comment]
Thanks Easton, this was mostly referring to other users. 

Solutions that already exist in game are of course helpful but sometimes an issue can be flooded with them which gradually becomes unhelpful. 

Thanks!

**Comment by epicEaston197** [comment]
> Thanks Easton, this was mostly referring to other users.
> 
> Solutions that already exist in game are of course helpful but sometimes an issue can be flooded with them which gradually becomes unhelpful.
> 
> Thanks!

yeah I just wanted to get the record straight on and get straight to the point
